client/question/components/LearnerViewDirective.js

```diff
@@ -871,12 +872,16 @@ tie.directive('learnerView', [function() {
           $scope.instructions = tasks[currentTaskIndex].getInstructions();
           $scope.previousInstructions = [];
           $scope.nextButtonIsShown = false;
+          EventHandlerService.initEventHandlerServiceState(
```

Comment:
I think you can just call this EventHandlerService.init()

Reply:
Done.